WSL/setup/environment.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Best practices for setting up a WSL 2 development environment
-description: A step-by-step guide to help you set up a WSL 2 development environment using Ubuntu, Visual Studio Code or Visual Studio, Git, Windows Credential Manager, and with recommended tutorials.
-keywords: Best practices, wsl, wsl 2, windows subsystem for linux, set up your dev environment with wsl 2, development environment, set up wsl 2
-ms.date: 09/27/2021
+title: Set up a WSL 2 development environment
+description: Set up a WSL 2 development environment using best practices from this set-by-step guide. Learn how to run Ubuntu, Visual Studio Code or Visual Studio, Git, Windows Credential Manager, MongoDB, MySQL, Docker remote containers and more.
+ms.date: 12/06/2021
 ms.topic: article
 ms.localizationpriority: medium
 no-loc: [Terminal]
+ms.custom: seo-windows-dev
 ---
 
-# Best practices for setting up a WSL development environment
+# Set up a WSL 2 development environment
```

Comment:
```suggestion
# Set up a WSL development environment
```

I'd rather keep it just as "WSL" since "WSL" is the product name not "WSL 2"

Reply:
851b17c